src/index.js

```diff
@@ -1,12 +1,54 @@
+import { html, LitElement } from '@polymer/lit-element';
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import GraphDisplayer from './GraphDisplayer';
 
-ReactDOM.render(<App />, document.getElementById('root'));
 
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+
+/**
+ * The LitElement class is an ultra-light wrapper for the W3C HTMLElement class.
+ * It is published by the Polymer team, and provides helpers for templating, binding and lifecycle functions.
+ * refer: https://github.com/Polymer/lit-element
+ *
+ * For info on 'slot' tag, refer to:
+ *  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot#Browser_compatibility
+ *  On browsers that do not natively support 'slot' elements, the webcomponentjs module will polyfill this.
+ *
+ *  This approach means that W3C standards can be used in consuming apps to pull in component apps,
+ *  without any framework lockin.  The aim is that shared webcomponents can use whichever framework
+ *  becomes popular over time, without consumers needing to know anything more than the webcomponent api.
+ *
+ */
+
+class GraphDisplayerWebComponent extends LitElement {
+  static get properties() {
+      return {
+          config: { type: Object },
+      };
+  }
+  constructor() {
+      super();
+      this.raiseGraphClicked = this.raiseGraphClicked.bind(this);
+  }
+  raiseGraphClicked(detail) {
+      this.dispatchEvent(new CustomEvent("graphClicked", {
+          bubbles: true,
+          cancelable: false,
+          composed: true,
+          detail
+      }));
+  }
```

Comment:
Is passing a method into the React component that actually triggers the event at the WebComponent layer the best way to do this?